test/mysql_handler/bench_test.go

```diff
@@ -0,0 +1,90 @@
+package main
+ import (
+	_ "github.com/go-sql-driver/mysql"
+	"database/sql"
+	"fmt"
+	"os"
+	"testing"
+)
+
+func getConnection() (*sql.DB, error) {
+	var ok bool
+	var address string
+
+	if address, ok = os.LookupEnv("BENCH_ADDRESS"); ok == false {
+	return nil, fmt.Errorf("%s is not set", "BENCH_ADDRESS")
+	}
+	address = fmt.Sprintf("test@%s", address)
+	db, err := sql.Open("mysql", "testuser:testpass@tcp(localhost:3306)/?tls=skip-verify")
```

Comment:
This line doesn't make use of the `address` variable above. I suspect it ought to be:
```
db, err := sql.Open("mysql", fmt.Sprintf("testuser:testpass@tcp(%s)/?tls=false", address))
```

Note that in the suggestion^ above `tls=false` as opposed to `tls=skip-verify`. It turns out `false` disables SSL, whereas `skip-verify` still uses SSL but does not verify. This would cause problems for testing with secretless.